python/salpytools/salpylib.py

```diff
@@ -33,19 +32,22 @@
 # fuction load_SALPYlib()
```

Comment:
This looks like it should be removed, together with line 35.

Reply:
Expanded note and removed comment.